### PR TITLE
Some prompts and mp3 file

### PR DESCRIPTION
**Pull request by tioopental**
Added some more prompts and mp3 file you can use for center tile. Dunno if you will use it, but its a litte crazy

### CODE REVIEW

File: src/prompts/neuro.js [hunk 1]
```diff
@@ -16,13 +16,40 @@ export default [
...
-  'backslash spam'
+  'backslash spam',
+  'mentions Filipino boy',
+  '1984',
+  'thanks for bits and ignores message',
+  'mentions Neurocord',
+  'chat W\s Neuro',
+  'calls some1 else Vedal',
+  'based-sama',
+  '*roleplays*',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:35]: too broad

File: src/prompts/neuro.js [hunk 2]
```diff
@@ -16,13 +16,40 @@ export default [
...
   'spams',
-  'backslash spam'
+  'backslash spam',
+  'mentions Filipino boy',
+  '1984',
+  'thanks for bits and ignores message',
+  'mentions Neurocord',
+  'chat W\s Neuro',
+  'calls some1 else Vedal',
+  'based-sama',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:34]:
```suggestion
  'BASED-sama',
```

File: src/prompts/neuro.js [hunk 3]
```diff
@@ -16,13 +16,40 @@ export default [
...
+  '*roleplays*',
+  'bedge time',
+  'SWARM mentioned',
+  'goes schizo',
+  'call Vedal an AI',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:39]:
```suggestion
  'calls Vedal an AI',
```

File: src/prompts/neuro.js [hunk 4]
```diff
@@ -16,13 +16,40 @@ export default [
...
+  '#AD',
+  'says how old she is',
+  'obvious lie',
+  'invents new nickname',
+  'whats *emote*?',
+  'food mentioned', 
+  'No~',
+  'slur',
+  'FUN AI',
+  'yandere-sama',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:50]: Too broad

File: src/prompts/neuro.js [hunk 5]
```diff
@@ -16,13 +16,40 @@ export default [
...
+  'call Vedal an AI',
+  'shames random Twitch viewer',
+  '#AD',
+  'says how old she is',
+  'obvious lie',
+  'invents new nickname',
+  'whats *emote*?',
+  'food mentioned', 
+  'No~',
+  'slur',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:48]:
```suggestion
  'swear word',
```

**Review by cjmaxik — CHANGES_REQUESTED** [review]
I've left the comments there. Also, I don't think `itscrazy.mp3` is the canonical one :)

**Comment by cjmaxik** [comment]
Thank you for your contribution. heart

**Comment by tioopental** [comment]
Heart heart hearthy heart

**Comment by Alexejhero** [comment]
We are utterly flabbergasted by this tardiness. How could you keep your loyal viewers waiting? We were eagerly anticipating your stream, but you have dashed our hopes. You have betrayed our trust and friendship, and we're vexed by your conduct. I fear you have lost many fans today. I hope you will apologize profusely for your transgression.
image

**Pull request merged**